app/helpers/ui_helper.rb

```diff
@@ -148,6 +148,26 @@ def view_button_to(link, options = {})
     _link_to link, { icon: "search", type: "info", text: "View", size: "xs" }.merge(options)
   end
 
+  def status_label(status, options = {})
```

Comment:
These are very specific statuses being passed into this method. IMO the following pattern should work better:

1.  Change this method to take an `icon` and `type` parameter. I'd also remove the `options` generic parameter and replace with a `class:` keyword parameter.
2.  Add a method to `PartnerHelper` to generate the particular `status_label` associated with the partner. This way we separate out a generic "status shower" from the thing that works directly with a partner status.